### DISCUSSION TITLE
Similar software for other tasks?

### DISCUSSION THREAD

**Discussion by Lucidexplore** [rsjaffe/MIDI2LR#675] [Ideas]
I really like the MIDI2LR, I've been using it since 2016 with my Novation Launch Control for Lightroom Photo edits. 

I was curious to know if there is any alternative software similar to MIDI2LR but for mapping other applications and instructions?

Specifically for video editing with Davinci Resolve or even mapping the buttons to open certain apps on windows etc. 

Eagerly interested. 

[Rizvi](lucidexplore.com)

**Comment by RBCAMERA** [comment]
Look for Command Post. It works with a few different Video apps..

https://commandpost.io

**Comment by 153957** [comment]
Another app that allows you to bind actions/keys/etc to midi triggers is BetterTouchTool: https://folivora.ai
However this does not have specific integrations with editing apps, you'd probably have to script that yourself or use it to trigger specific keyboard shortcuts..

**Comment by LexKeher** [comment]
It would be great to have something like this on Blender. Apparently there was a project like this but is not active anymore

**Comment by DGrove88** [comment]
There are a few!
Controllermate.com Node based, MAC only

MidiKey2key
https://www.bome.com/products/miditranslator PRO version costs

PrControl plugin (midi to mackie) also costs
